### PR TITLE
Validate InlineArrayAttribute applications on types in source.

### PR DESCRIPTION
**Pull request by AlekseyTs**
https://github.com/dotnet/csharplang/blob/main/proposals/inline-arrays.md#validation-of-the-inlinearrayattribute-applications

### CODE REVIEW

**Comment by AlekseyTs** [comment]
@cston, @jjonescz, @dotnet/roslyn-compiler Please review

File: src/Compilers/CSharp/Test/Semantic/Semantics/InlineArrayTests.cs [hunk 1]
```diff
@@ -161,22 +161,62 @@ void verify(ModuleSymbol m)
         public void InlineArrayType_05_WrongLength()
         {
             var src = @"
+#pragma warning disable CS0169 // The field 'Buffer._element0' is never used
+
 [System.Runtime.CompilerServices.InlineArray(0)]
```
Comment (jjonescz) [inline src/Compilers/CSharp/Test/Semantic/Semantics/InlineArrayTests.cs:166]: Consider also testing a custom attribute definition without parameters applied as `[System.Runtime.CompilerServices.InlineArray]`.

**Review by jjonescz — APPROVED** [review]

**Comment by AlekseyTs** [comment]
@cston, @dotnet/roslyn-compiler For the second review

File: src/Compilers/CSharp/Test/Semantic/Semantics/InlineArrayTests.cs (hunk 1 above)
Comment (AlekseyTs) [reply src/Compilers/CSharp/Test/Semantic/Semantics/InlineArrayTests.cs:166]: >Consider also testing a custom attribute definition without parameters applied ...

Added

**Review by cston — APPROVED** [review]

**Pull request merged**